tests/php/sync/test_class.jetpack-sync-comments.php

```diff
@@ -47,6 +47,27 @@ public function test_update_comment() {
 		$this->assertEquals( "foo bar baz", $remote_comment->comment_content );
 	}
 
+	public function test_modify_comment_contents() {
+
+		//Confirm that 'modify_comment' action is set after changing comment contents
+		$this->comment->comment_content = "foo bar baz";
```

Comment:
lets expand this to test also the changes to the name, email, website and content